Dockerfile

```diff
@@ -14,12 +14,16 @@ RUN <<EOR
 	fi
 
 	apt-get update
+	export DEBIAN_FRONTEND=noninteractive
 	apt-get install -y --no-install-recommends \
 		gcc \
+		git \
 		librsvg2-bin \
+		libxfixes3 \
 		linux-headers-generic \
 		python3-dev \
 		python3-setuptools \
+		python3-usb \
```

Comment:
libusb1 is installed later with pip, is this necessary?

Reply:
If removed, Docker build fails on python tests with
```
OSError: libusb-1.0.so: cannot open shared object file: No such file or directory
```